src/Ilios/CoreBundle/Handler/PendingUserUpdateHandler.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Ilios\CoreBundle\Handler;
+
+use Symfony\Component\Form\FormFactoryInterface;
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Ilios\CoreBundle\Exception\InvalidFormException;
+use Ilios\CoreBundle\Form\Type\PendingUserUpdateType;
+use Ilios\CoreBundle\Entity\Manager\PendingUserUpdateManager;
+use Ilios\CoreBundle\Entity\PendingUserUpdateInterface;
+
+/**
+ * Class PendingUserUpdateHandler
+ * @package Ilios\CoreBundle\Handler
+ */
+class PendingUserUpdateHandler extends PendingUserUpdateManager
+{
+    /**
+     * @var FormFactoryInterface
+     */
+    protected $formFactory;
+
+    /**
+     * @param Registry $em
+     * @param string $class
+     * @param FormFactoryInterface $formFactory
+     */
+    public function __construct(Registry $em, $class, FormFactoryInterface $formFactory)
+    {
+        $this->formFactory = $formFactory;
+        parent::__construct($em, $class);
+    }
+
+    /**
+     * @param array $parameters
+     *
+     * @return PendingUserUpdateInterface
+     */
+    public function post(array $parameters)
+    {
+        $pendingUserUpdate = $this->createPendingUserUpdate();
+
+        return $this->processForm($pendingUserUpdate, $parameters, 'POST');
+    }
+
+    /**
+     * @param PendingUserUpdateInterface $pendingUserUpdate
+     * @param array $parameters
+     *
+     * @return PendingUserUpdateInterface
+     */
+    public function put(
+        PendingUserUpdateInterface $pendingUserUpdate,
+        array $parameters
+    ) {
+        return $this->processForm(
+            $pendingUserUpdate,
+            $parameters,
+            'PUT'
+        );
+    }
+
+    /**
+     * @param PendingUserUpdateInterface $pendingUserUpdate
```

Comment:
Looks like we need to pull that out of all the handlers, but it's not doing any harm right now.